app/src/main/java/com/mk/tasky/core/data/TaskyRepositoryImpl.kt

```diff
@@ -1,20 +1,20 @@
-package com.mk.tasky.authentication.data
+package com.mk.tasky.core.data
 
 import com.mk.tasky.authentication.data.mapper.toDomain
-import com.mk.tasky.authentication.data.remote.AuthenticationApi
 import com.mk.tasky.authentication.data.remote.dto.ErrorResponseDto
 import com.mk.tasky.authentication.data.remote.dto.LoginBodyDto
 import com.mk.tasky.authentication.data.remote.dto.RegistrationBodyDto
 import com.mk.tasky.authentication.data.remote.exceptions.LoginException
-import com.mk.tasky.authentication.domain.AuthenticationRepository
-import com.mk.tasky.authentication.domain.models.LoggedUser
+import com.mk.tasky.core.data.remote.TaskyApi
+import com.mk.tasky.core.domain.model.LoggedUser
+import com.mk.tasky.core.domain.repository.TaskyRepository
 import com.squareup.moshi.Moshi
 import retrofit2.HttpException
 import java.util.concurrent.CancellationException
 
-class AuthenticationRepositoryImpl(
-    private val api: AuthenticationApi
-) : AuthenticationRepository {
+class TaskyRepositoryImpl(
+    private val api: TaskyApi
+) : TaskyRepository {
```

Comment:
Why did you rename this? `AuthenticationRepository` is much more expressive than `TaskyRepository` (which screams to me "this is the repo where we put in all the stuff that's shared" 😆 )

Reply:
Yeah you are right.
Fixed!